cmd/clusterctl/client/cluster/mover.go

```diff
@@ -350,7 +350,7 @@ func setClusterPause(proxy Proxy, clusters []*node, value bool, dryRun bool) err
 		if err := retryWithExponentialBackoff(setClusterPauseBackoff, func() error {
 			return patchCluster(proxy, cluster, patch)
 		}); err != nil {
-			return err
+			return errors.Wrapf(err, "error setting Cluster.Spec.Paused=%t", value)
```

Comment:
What's the convention here? should we standardise on using core package i.e `fmt.Errorf("blah: %w", err)`

Reply:
I"m not sure. I just used what was used in the code around here (e.g. l.378).

@fabriziopandini wdyt?

Reply:
We are using consistently `github.com/pkg/errors`, let's stick to this